install.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh
```

Comment:
I would use bash to be more flexible. I believe `${VERSION//./-}` is already not POSIX.